CONTRIBUTING.md

```diff
@@ -1,109 +1,140 @@
 # Welcome contributors to Alva
+
 💓 First of all, we'd love for you to contribute to our source code. 💓
 
 ## Code of Conduct
+
 We believe in a welcoming and inclusive environment / community, that's why we enforce our Code of Conduct.
 For more information -> [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md).
 
-
 ## Table of Contents
-- [Link collection](#short-collection-of-links-to-important-resources)
-- [Setup for contributors](#setup-for-contributors)
-- [Issues](#issues)
-	- [Good first issue for Contributors](#good-first-issue)
-	- [Label structure](#labels)
-	- [Working on issues](#working-on-issues)
-- [Style Guide / Coding conventions](#style-guide--coding-conventions)
-	- [Coding conventions & guidelines](#coding-conventions--guidelines)
 
+* [Link collection](#short-collection-of-links-to-important-resources)
+* [Setup for contributors](#setup-for-contributors)
+* [Issues](#issues)
+  * [Good first contribution](#good-first-issue)
+  * [Label structure](#labels)
+  * [Working on issues](#working-on-issues)
+* [Style Guide / Coding conventions](#style-guide--coding-conventions)
+  * [Coding conventions & guidelines](#coding-conventions--guidelines)
 
 ## Short collection of Links to Important Resources
-   * bugs / feature requests: https://github.com/meetalva/alva/issues
-   * website: https://meetalva.io
-   * sprint board: https://github.com/meetalva/alva/projects/2
-   * backlog board: https://github.com/meetalva/alva/projects/3
 
+* bugs / feature requests: https://github.com/meetalva/alva/issues
+* website: https://meetalva.io
+* sprint board: https://github.com/meetalva/alva/projects/2
+* backlog board: https://github.com/meetalva/alva/projects/3
 
 ## Setup for contributors
+
 To add a contribution to Alva (improve its features, fix issues),
-get the source of the application itself, and if you don´t already have a Alva compatible styleguide, the kickstart styleguide (designkit):
+get the source of the application itself, and if you don't already have a Alva compatible styleguide, the kickstart styleguide (designkit):
 
 1. Clone the [alva](https://github.com/meetalva/alva) repository:
+
 ```shell
 git clone git@github.com:meetalva/alva.git
 ```
+
 2. Run in the main (Alva) repository:
+
 ```shell
 npm i && npm start
 ```
 
 3. Now open a compatible styleguide.
 
-If you don´t have any, follow the next steps
+If you don't have any, follow the next steps:
```

Comment:
Maybe we should specify: "If you don’t have any compatible styleguide [...]" and add a small space above the section?

Reply:
Done.